lib/gateway/stat_gateway.rb

```diff
@@ -4,31 +4,39 @@ def signups
       today: signups_today.count,
       total: signups_total.count,
       sms_today: sms_signups_today.count,
-      sms_total: sms_signups_total.count
+      sms_total: sms_signups_total.count,
+      email_today: email_signups_today.count,
+      email_total: email_signups_total.count
     }
```

Comment:
Side note: I like word `total` on a hash key. Makes sense because that's a number. I dislike it as a suffix for a method name... because that method returns a query, not a *total*-of-anything value. May change it to `all` later...

Reply:
Why not make the change now, i'd argue that `cumulative_email_signups` is more readable.

Reply:
Where's the accumulation?